docs/bosh/README.md

```diff
@@ -123,259 +155,263 @@ Now you have the infrastructure ready to deploy a BOSH director.
-          release: bosh
-        - name: director
-          release: bosh
-        - name: health_monitor
-          release: bosh
-        - name: google_cpi
-          release: bosh-google-cpi
-
-      resource_pool: vms
-      persistent_disk_pool: disks
-
-      networks:
-        - name: private
-          static_ips: [<%= ENV['base_ip'].split('.').tap{|i| i[-1] = i[-1].to_i + 6 }.join('.') %>]
-          default:
-            - dns
-            - gateway
-
-      properties:
-        nats:
-          address: 127.0.0.1
-          user: nats
-          password: nats-password
-
-        postgres: &db
-          listen_address: 127.0.0.1
-          host: 127.0.0.1
-          user: postgres
-          password: postgres-password
-          database: bosh
-          adapter: postgres
-
-        dns:
-          address: <%= ENV['base_ip'].split('.').tap{|i| i[-1] = i[-1].to_i + 6 }.join('.') %>
-          domain_name: microbosh
-          db: *db
-          recursor: 169.254.169.254
-
-        blobstore:
-          address: <%= ENV['base_ip'].split('.').tap{|i| i[-1] = i[-1].to_i + 6 }.join('.') %>
-          port: 25250
-          provider: dav
-          director:
-            user: director
-            password: director-password
-          agent:
-            user: agent
-            password: agent-password
-
-        director:
-          address: 127.0.0.1
-          name: micro-google
-          db: *db
-          cpi_job: google_cpi
-          user_management:
-            provider: local
-            local:
-              users:
-                - name: admin
-                  password: admin
-                - name: hm
-                  password: hm-password
-        hm:
-          director_account:
-            user: hm
-            password: hm-password
-          resurrector_enabled: true
-
-        google: &google_properties
-          project: <%= ENV['project_id'] %>
-
-        agent:
-          mbus: nats://nats:nats-password@<%= ENV['base_ip'].split('.').tap{|i| i[-1] = i[-1].to_i + 6 }.join('.') %>:4222
-          ntp: *ntp
-          blobstore:
-             options:
-               endpoint: http://<%= ENV['base_ip'].split('.').tap{|i| i[-1] = i[-1].to_i + 6 }.join('.') %>:25250
-               user: agent
-               password: agent-password
-
-        ntp: &ntp
-          - 169.254.169.254
-
-  cloud_provider:
-    template:
-      name: google_cpi
-      release: bosh-google-cpi
-
-    ssh_tunnel:
-      host: <%= ENV['base_ip'].split('.').tap{|i| i[-1] = i[-1].to_i + 6 }.join('.') %>
-      port: 22
-      user: bosh
-      private_key: <%= ENV['ssh_key_path'] %>
-
-    mbus: https://mbus:mbus-password@<%= ENV['base_ip'].split('.').tap{|i| i[-1] = i[-1].to_i + 6 }.join('.') %>:6868
-
-    properties:
-      google: *google_properties
-      agent: {mbus: "https://mbus:mbus-password@0.0.0.0:6868"}
-      blobstore: {provider: local, path: /var/vcap/micro_bosh/data/cache}
-      ntp: *ntp
-  ```
+   ```
```

Comment:
Great question. I suspect this hasn't been updated since we started cloning this repo into /share as part of provisioning the bastion (and use it for [CF deployments](https://github.com/cloudfoundry-incubator/bosh-google-cpi-release/blob/master/docs/cloudfoundry/README.md#steps-to-perform-in-bosh-bastion))